dotnet/src/SemanticKernel/CoreSkills/SemanticFunctionConstants.cs

```diff
@@ -10,11 +10,11 @@ internal static class SemanticFunctionConstants
 1. From a <goal> create a <plan> as a series of <functions>.
 2. Use only the [AVAILABLE FUNCTIONS] - do not create new functions, inputs or attribute values.
```

Comment:
>AVAILABLE FUNCTIONS [](http://example.com/codeflow?start=17&length=19)

Can we also return the relevancy score for each of the functions? Or have a debug flag that returns the relevancy score for each step if you don't want to include this in the plan data by default?

Reply:
For now, I think I'll go ahead and add a debug log at minimum. I'll give this some more thought/design with our planning.

Reply:
Ok, I think a flag would be a little more helpful so that can still see the whole plan to modify/render with the relevancy score